src/firefly/js/api/ApiExpandedView.jsx

```diff
@@ -23,13 +25,24 @@ export function ApiExpandedView() {
 
     const expandType = LO_VIEW.get(expanded) ?? LO_VIEW.none;
     if (expandType === LO_VIEW.none) {
-        document.body.style.overflow= current.saveOverflow;
+        if (isDefined(current.saveOverflow)) document.body.style.overflow= current.saveOverflow;
         return false;
     }
 
-    const {chartId} = expandType === LO_VIEW.xyPlots ? getExpandedChartProps() : {};
     current.saveOverflow= document.body.style.overflow;
     document.body.style.overflow= 'hidden';
```

Comment:
Instead of saving and then resetting `body.overflow`, maybe we should not set it begin with.

Reply:
We had to set it when expanded to make the expand look correct.  I don't remember but I think there was a scroll bar on some of the web pages that did not appear to do anything. This is just putting it back to what ever the web page had before it was expanded. I don't  know of any other solution here.

Reply:
Ah.  I see what you mean now.  JoyUI is doing the same thing, except they are modify the `html` element.  Ignore this comment.